composer.json

```
@@ -15,10 +15,10 @@
     },
     "require-dev" : {
         "squizlabs/php_codesniffer" : "2.9.2",
-        "phpunit/phpunit" : "6.5.*",
+        "phpunit/phpunit" : "^8",
         "facebook/webdriver" : "dev-master",
         "phpmd/phpmd": "^2.6",
-        "phan/phan": ">1.2.2"
+        "phan/phan": ">2.2.4"
```

```suggestion
        "phan/phan": ">2.2.11"
```

2.2.12 is the latest version as of now.